pkg/volume/awsebs/attacher_test.go

```diff
@@ -128,15 +128,16 @@ func TestAttachDetach(t *testing.T) {
 	}
 
 	for _, testcase := range tests {
-		testcase.t = t
-		device, err := testcase.test(&testcase)
-		if err != testcase.expectedError {
-			t.Errorf("%s failed: expected err=%q, got %q", testcase.name, testcase.expectedError.Error(), err.Error())
-		}
-		if device != testcase.expectedDevice {
-			t.Errorf("%s failed: expected device=%q, got %q", testcase.name, testcase.expectedDevice, device)
-		}
-		t.Logf("Test %q succeeded", testcase.name)
+		t.Run(testcase.name, func(t *testing.T) {
+			testcase.t = t
+			device, err := testcase.test(&testcase)
+			if err != testcase.expectedError {
+				t.Errorf("failed: expected err=%q, got %q", testcase.expectedError.Error(), err.Error())
```

Comment:
so you don't want to log testcase name here?

Reply:
When divided into subsets, the name of the subset is printed at the beginning. 
such as "TestAttachDetach/Attach_Positive".